axelor-sale/src/main/resources/views/Charts.xml

```diff
@@ -705,6 +705,55 @@
 		<category key="_industry_sector" type="text" title="Industry Sector"/>
 		<series key="_amount" type="donut" title="Turnover"/>
 	</chart>
+
+	<chart name="chart.margin.rate.per.customer" title="Average margin rate per customer" onInit="action.sale.chart.set.date">
+		<search-fields>
+			<field name="fromDate" title="From Date" type="date" x-required="true"/>
+			<field name="toDate" title="To Date" type="date" x-required="true"/>
+		</search-fields>
+		<dataset type="jpql"><![CDATA[
+			SELECT
```

Comment:
Is this possible here to use rpc to call a method in PartnerService ? This would avoid duplicating the query.
If this is not possible the current way it is done is fine.